src/components/side_panel/chart/combo_chart/combo_chart_design_panel.xml

```diff
@@ -3,7 +3,16 @@
     <GeneralDesignEditor t-props="props">
       <t t-set-slot="general-extension">
         <ChartLegend t-props="props"/>
-        <ChartShowValues t-props="props"/>
+        <ChartShowValues t-props="props">
+          <t t-set-slot="values-extension">
+            <Checkbox
+              name="'showDataMarkers'"
```

Comment:
could add the option to radar charts also

Reply:
And if you add it to radar chart, that's 3 places with the same checkbox. Maybe create a dedicated component at this point ?